apollo-client/src/main/java/com/ctrip/framework/apollo/internals/AbstractConfig.java

```diff
@@ -493,11 +495,11 @@ private boolean isConfigChangeListenerInterested(ConfigChangeListener configChan
   List<ConfigChange> calcPropertyChanges(String namespace, Properties previous,
                                          Properties current) {
     if (previous == null) {
-      previous = new Properties();
+      previous =  PropertiesFactory.getPropertiesObject();
```

Comment:
I think we need to use `ApolloInjector` to get the PropertiesFactory instance so that it is testable.

Reply:
btw, there is one more extra space before `PropertiesFactory`

Reply:
Already changed in latest commits. Please tell me  if  I use `ApolloInjector` in a wrong way.

Reply:
It looks OK.